src/app/page.tsx

```diff
@@ -1,12 +1,14 @@
 "use client";
 
 import ExampleButton from "@components/ExampleButton";
```

Comment:
nit: Remove unused imports

Reply:
Like Mr PM said looks really good guys, in general we wanna remove unused imports/variables wherever possible to make the code as clean as possible. It's currently failing to deploy on Vercel because eslint is detecting the unused import while building. You can check if it's building correctly before making a PR by using the "npm run build" command